packages/salesforce-adapter/src/filters/formula_utils/utils.ts

```diff
@@ -0,0 +1,33 @@
+/*
+*                      Copyright 2022 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+
+const STANDARD_RELATIONSHIP_ID_NAME = 'Id'
+const DOLLAR_SIGN = '$'
+
+export const $ = (value: string): string => value.toUpperCase()
+export const parts = (value: string): string[] => value.split('.')
```

Comment:
nit: this is the reverse of `createApiName` and should reflect it

```suggestion
export const splitApiName  = (value: string): string[] => value.split('.')
```